### PR TITLE
Fix Build.cmd to include examples binaries in release zip file

### PR DESCRIPTION
**Pull request by hebinhuang**
The release zip file includes source files under "examples" folder. the fix is to include examples binaries in release zip file.

Refer to #289

### CODE REVIEW

File: build/Build.cmd [hunk 1]
```diff
@@ -142,16 +168,16 @@ xcopy /e /y "%CMDHOME%\..\scripts"  "%SPARKCLR_HOME%\scripts\"
 pushd "%CMDHOME%"
 if not exist ".\target" (mkdir .\target)
 
+set ProjectVersion=1.6.0-preview-3
```
Comment (skaarthik) [inline build/Build.cmd]: why is this needed?

File: build/Build.cmd [hunk 2]
```diff
@@ -142,16 +168,16 @@ xcopy /e /y "%CMDHOME%\..\scripts"  "%SPARKCLR_HOME%\scripts\"
...
+
 if not defined ProjectVersion (
     powershell -f .\localmode\zipdir.ps1 -dir "%SPARKCLR_HOME%" -target ".\target\runtime.zip"
     goto :distdone
 )
 
-set SPARKCLR_NAME=spark-clr_2.10-%ProjectVersion%
```
Comment (skaarthik) [inline build/Build.cmd]: ...and this change?

File: build/Build.cmd (hunk 1 above)
Comment (hebinhuang) [inline build/Build.cmd]: It looks like the merge between my private branch and master has problems. this line is just for my test propose. Actually, I have removed the line before commit. Not sure why it came back.

Let me do a clean branch again.
